bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/pageobjects/touchui/DialogConfigurer.java

```diff
@@ -80,6 +80,7 @@ public DialogField getDialogField(WebElement parentElement, String label, String
    */
   public DialogField getDialogField(WebElement parentElement, String type) {
     WebElement scope = parentElement.findElement(By.tagName("input"));
+    scope.clear();
```

Comment:
The getDialogFIeld method should't clear any field. I would suggest to clear a field in setValue method in field related class, e.g.  Image from com.cognifide.qa.bb.aem.touch.pageobjects.touchui.dialogfields